babel.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = api => {
+    const isTest = api.env('test');
+    const presets = [];
+    const plugins = [];
+
+    if (!isTest) {
+        api.cache(true);
```

Comment:
Might be worth checking to see if we actually need this

Reply:
Pretty sure it helps babel run a little faster (as only runs on changed files this way)